src/shared-components/progressBar/index.tsx

```diff
@@ -4,18 +4,34 @@ import PropTypes from 'prop-types';
 import { COLORS, PROGRESS_BAR_STATUS } from '../../constants';
 import { OuterContainer, InnerBar } from './style';
 
+export type ProgressBarStatusType =
+  | typeof PROGRESS_BAR_STATUS.error
+  | typeof PROGRESS_BAR_STATUS.loading
+  | typeof PROGRESS_BAR_STATUS.success;
+
+type ProgressBarProps = {
+  backgroundColor?: string;
+  barColor?: string;
+  height?: number;
+  loadingTime?: string;
+  status: ProgressBarStatusType;
+  // ...rest type
+  [key: string]: any;
+};
+
 const ProgressBar = ({
-  backgroundColor,
-  barColor,
-  height,
-  loadingTime,
+  backgroundColor = COLORS.background,
```

Comment:
We can just go ahead and remove the `defaultProps` and rely on ES6 defaults alone. I think that should be our preference going forward.

Reply:
eslint will protest that non required proptypes requires a default Proptype

Reply:
@daigof We can eliminate that rule for function components by changing the `eslint` rules:

```
"react/require-default-props": [
      "error",
      { "ignoreFunctionalComponents": true }
    ],
```

Reply:
done. nice, Thanks for this standarization, Ill start applying in Radiance